cryptus-app/__tests__/snapshot/market_overview/sort_button/__snapshots__/sort_button.snapshot.test.tsx.snap

```diff
@@ -2,13 +2,40 @@
 
 exports[`<SortButton /> should display properly 1`] = `
 <div>
-  <div>
-    <img
-      src="menu.svg"
-    />
-    <div>
-      Sort
-    </div>
-  </div>
+  <button
+    aria-haspopup="true"
+    className="MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeSmall MuiButton-textSizeSmall MuiButtonBase-root  css-1knaqv7-MuiButtonBase-root-MuiButton-root"
+    disabled={false}
+    id="basic-button"
+    onBlur={[Function]}
+    onClick={[Function]}
+    onContextMenu={[Function]}
+    onDragLeave={[Function]}
+    onFocus={[Function]}
+    onKeyDown={[Function]}
+    onKeyUp={[Function]}
+    onMouseDown={[Function]}
+    onMouseLeave={[Function]}
+    onMouseUp={[Function]}
+    onTouchEnd={[Function]}
+    onTouchMove={[Function]}
+    onTouchStart={[Function]}
+    tabIndex={0}
+    type="button"
+  >
+    <span
+      className="MuiButton-startIcon MuiButton-iconSizeSmall css-y6rp3m-MuiButton-startIcon"
+    >
+      <div
+        className="MuiAvatar-root MuiAvatar-circular css-1wlk0hk-MuiAvatar-root"
+      >
```

Comment:
a div cannot be inside of a span

Reply:
c'est les snapshots tests c'est auto généré

Reply:
rip